developer_manual/core/ocs-share-api.rst

```diff
@@ -129,3 +130,64 @@ Statuscodes:
 * 400 - wrong or no update parameter given
 * 403 - public upload disabled by the admin
 * 404 - couldn't update share
+
+
+Federated Cloud Shares
+======================
```

Comment:
What's the correct way to do cross-manual references @carlaschroder 
The following seems to be not working, although it matches the expected path:
```rst
:doc:`../../admin_manual/configuration_files/federated_cloud_sharing_configuration`
```